app/views/pages/connect.html.erb

```diff
@@ -12,7 +12,7 @@
   <section class="connect">
   <div class="connectionTypes">
     <a href="/saved_query_doc" class="connectionCard">
-      <h4>Browser</h4>
+      <h4>Click here to run queries on the AACT data from your web browser</h4>
```

Comment:
This should be "Browser".

Reply:
It says "In connect change the description for “Browser” to “Click here to run queries on the AACT data from your web browser”"

Reply:
Fixed, thank you!